config.yml

```diff
@@ -82,6 +82,7 @@ params:
     js_bundle_hash:   "sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN"
     popper:           "https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
     popper_hash:      "sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+    popper_esm:       "https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/esm/popper.min.js"
```

Comment:
@XhmikosR Is this solution OK for you as well? Or would you prefer another way by keeping only `cdn.popper` and to replace "umd" by "esm" on the fly thanks to a Hugo pipe or something where needed?

Reply:
Saving this for later if needed.